.github/dependabot.yml

```diff
@@ -23,9 +23,9 @@ updates:
   - package-ecosystem: "cargo"
     directory: "/"
     schedule:
-      interval: "weekly"
```

Comment:
Your title says increasing the interval, you have done the opposite.

IMHO we should keep the current schedule, is there any reason to change it?

Reply:
Indeed I made the mistake in the title (corrected, thanks!).
We have talked about it with @BalduinLandolt last week. The current state of this service/app is kind of static/simple - no authentication, no user profile, no fragile data etc. Therefore IMO it does not make sense for more frequent dependency updates. Especially if the new version is most probably going to be released on demand only and looking back, it was even less frequent than once per month.
Once we see there is necessity for the interval increasement, it could be done.
On top of that, there is no option for PR grouped PR title created by Dependabot, which will create confusion in the release notes. Which IMO is fine, until one would need to find out in which release specific dependency was bumped. Probably not common case, but still something to keep in mind.

Reply:
Your "reasons" do not convince me. I do think merging some small PR every other week is not a problem.
That we we have the latests updates once we release on demand much quicker, which I deem to be more important.

Reply:
It is simply because my "reasons" are REASONS when you try to use them.